kv/kvserver: TestRaftReceiveQueuesEnforceMaxLenConcurrency failed

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162005?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162005?buildTab=artifacts#/) on master @ [a55295b0222cf4b8aed60315603c0b627c5ee9da](https://github.com/cockroachdb/cockroach/commits/a55295b0222cf4b8aed60315603c0b627c5ee9da):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m28s)

```
Stack: 

```
goroutine 74136 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-45544

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162229?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162229?buildTab=artifacts#/) on master @ [a55295b0222cf4b8aed60315603c0b627c5ee9da](https://github.com/cockroachdb/cockroach/commits/a55295b0222cf4b8aed60315603c0b627c5ee9da):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m25s)

```
Stack: 

```
goroutine 76831 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162245?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162245?buildTab=artifacts#/) on master @ [a55295b0222cf4b8aed60315603c0b627c5ee9da](https://github.com/cockroachdb/cockroach/commits/a55295b0222cf4b8aed60315603c0b627c5ee9da):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m28s)

```
Stack: 

```
goroutine 79383 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162355?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162355?buildTab=artifacts#/) on master @ [603ff88e54d3e3f6e49b0f673abd5ec564bf418b](https://github.com/cockroachdb/cockroach/commits/603ff88e54d3e3f6e49b0f673abd5ec564bf418b):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m30s)

```
Stack: 

```
goroutine 76701 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162939?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18162939?buildTab=artifacts#/) on master @ [603ff88e54d3e3f6e49b0f673abd5ec564bf418b](https://github.com/cockroachdb/cockroach/commits/603ff88e54d3e3f6e49b0f673abd5ec564bf418b):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m28s)

```
Stack: 

```
goroutine 78406 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18163242?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18163242?buildTab=artifacts#/) on master @ [603ff88e54d3e3f6e49b0f673abd5ec564bf418b](https://github.com/cockroachdb/cockroach/commits/603ff88e54d3e3f6e49b0f673abd5ec564bf418b):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m29s)

```
Stack: 

```
goroutine 75611 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18164261?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18164261?buildTab=artifacts#/) on master @ [fada0424873ba279e5cfeb193d7d9d0b760693e3](https://github.com/cockroachdb/cockroach/commits/fada0424873ba279e5cfeb193d7d9d0b760693e3):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m57s)

```
Stack: 

```
goroutine 79782 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18164526?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18164526?buildTab=artifacts#/) on master @ [fada0424873ba279e5cfeb193d7d9d0b760693e3](https://github.com/cockroachdb/cockroach/commits/fada0424873ba279e5cfeb193d7d9d0b760693e3):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m28s)

```
Stack: 

```
goroutine 74126 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18165891?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18165891?buildTab=artifacts#/) on master @ [ada0ea729db30bcf32dc9f0b015086f2a3460a4b](https://github.com/cockroachdb/cockroach/commits/ada0ea729db30bcf32dc9f0b015086f2a3460a4b):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m29s)

```
Stack: 

```
goroutine 75880 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18166657?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18166657?buildTab=artifacts#/) on master @ [ada0ea729db30bcf32dc9f0b015086f2a3460a4b](https://github.com/cockroachdb/cockroach/commits/ada0ea729db30bcf32dc9f0b015086f2a3460a4b):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m59s)

```
Stack: 

```
goroutine 80134 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18169637?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18169637?buildTab=artifacts#/) on master @ [559007477c122779e5b9a843b9362d9a304b320e](https://github.com/cockroachdb/cockroach/commits/559007477c122779e5b9a843b9362d9a304b320e):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m29s)

```
Stack: 

```
goroutine 76241 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18170251?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18170251?buildTab=artifacts#/) on master @ [047d1bed5ed7c0f3dcab6d564a62cce3e24186df](https://github.com/cockroachdb/cockroach/commits/047d1bed5ed7c0f3dcab6d564a62cce3e24186df):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m23s)

```
Stack: 

```
goroutine 80085 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18171328?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18171328?buildTab=artifacts#/) on master @ [e894ec790bc7952c7937d641d5c7641b20bd5f4f](https://github.com/cockroachdb/cockroach/commits/e894ec790bc7952c7937d641d5c7641b20bd5f4f):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m15s)

```
Stack: 

```
goroutine 82497 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18172473?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18172473?buildTab=artifacts#/) on master @ [40a187c81853246ada4525a1d85abd6464e55fd1](https://github.com/cockroachdb/cockroach/commits/40a187c81853246ada4525a1d85abd6464e55fd1):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m59s)

```
Stack: 

```
goroutine 79205 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18173069?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18173069?buildTab=artifacts#/) on master @ [891961468bd8de2a34503c5052b71c3c81220328](https://github.com/cockroachdb/cockroach/commits/891961468bd8de2a34503c5052b71c3c81220328):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m1s)

```
Stack: 

```
goroutine 74864 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18174115?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18174115?buildTab=artifacts#/) on master @ [42593c0e6aa1a4a7d74b58b3849efc6e502172d6](https://github.com/cockroachdb/cockroach/commits/42593c0e6aa1a4a7d74b58b3849efc6e502172d6):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m59s)

```
Stack: 

```
goroutine 83823 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18174959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18174959?buildTab=artifacts#/) on master @ [ced8f94a3ef600c1a5408119842b9e93eadcd87a](https://github.com/cockroachdb/cockroach/commits/ced8f94a3ef600c1a5408119842b9e93eadcd87a):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m55s)

```
Stack: 

```
goroutine 85125 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18175842?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18175842?buildTab=artifacts#/) on master @ [3878bf6a9173e1654985c8fafc198b82fc552eac](https://github.com/cockroachdb/cockroach/commits/3878bf6a9173e1654985c8fafc198b82fc552eac):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m28s)

```
Stack: 

```
goroutine 80316 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18177048?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18177048?buildTab=artifacts#/) on master @ [a4d45a44e68ecd57fa76c2a48c19f271e57a63b0](https://github.com/cockroachdb/cockroach/commits/a4d45a44e68ecd57fa76c2a48c19f271e57a63b0):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (13m52s)

```
Stack: 

```
goroutine 81852 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18177506?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18177506?buildTab=artifacts#/) on master @ [57a7b119a00c0559b8de83aed63deeaf67d115e5](https://github.com/cockroachdb/cockroach/commits/57a7b119a00c0559b8de83aed63deeaf67d115e5):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m29s)

```
Stack: 

```
goroutine 80514 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18178690?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18178690?buildTab=artifacts#/) on master @ [6448bef4788eaefad238f88be4a350ee53e2d256](https://github.com/cockroachdb/cockroach/commits/6448bef4788eaefad238f88be4a350ee53e2d256):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m30s)

```
Stack: 

```
goroutine 76974 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18178775?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18178775?buildTab=artifacts#/) on master @ [81950d14d0b77d9bbedbeaff3c11fe4bbd169116](https://github.com/cockroachdb/cockroach/commits/81950d14d0b77d9bbedbeaff3c11fe4bbd169116):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m29s)

```
Stack: 

```
goroutine 69510 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18179076?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18179076?buildTab=artifacts#/) on master @ [81950d14d0b77d9bbedbeaff3c11fe4bbd169116](https://github.com/cockroachdb/cockroach/commits/81950d14d0b77d9bbedbeaff3c11fe4bbd169116):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m1s)

```
Stack: 

```
goroutine 78078 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

kv/kvserver.TestRaftReceiveQueuesEnforceMaxLenConcurrency [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18179165?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/18179165?buildTab=artifacts#/) on master @ [81950d14d0b77d9bbedbeaff3c11fe4bbd169116](https://github.com/cockroachdb/cockroach/commits/81950d14d0b77d9bbedbeaff3c11fe4bbd169116):

Fatal error:

```
panic: test timed out after 15m0s
running tests:
	TestRaftReceiveQueuesEnforceMaxLenConcurrency (14m27s)

```
Stack: 

```
goroutine 77218 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2366 +0x30c
created by time.goFunc
	GOROOT/src/time/sleep.go:177 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestRaftReceiveQueuesEnforceMaxLenConcurrency
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRaftReceiveQueuesEnforceMaxLenConcurrency.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/kv/kvserver/store_raft_test.go
+++ b/pkg/kv/kvserver/store_raft_test.go
@@ -270,7 +270,7 @@ func TestRaftReceiveQueuesEnforceMaxLenConcurrency(t *testing.T) {
 	}
 	// Iterate and set different values of enforceMaxLen.
 	enforceMaxLen := false
-	for i := 0; i < 200; i++ {
+	for i := 0; i < 25; i++ {
 		// NB: SetEnforceMaxLen runs concurrently with LoadOrCreate calls.
 		qs.SetEnforceMaxLen(enforceMaxLen)
 		// Exclude all LoadOrCreate calls while checking is happening.

